uPortal-webapp/src/test/java/org/apereo/portal/portlet/dao/jpa/JpaMarketplaceRatingDaoTest.java

```diff
@@ -279,4 +286,137 @@ public Object call() throws Exception {
                     }
                 });
     }
+
+    @Test
+    public void testGetRatingsLastXDays() {
+        // Easy Test Can we make every combination of portlet and user rating
+        this.execute(
+                new Callable<Object>() {
+                    @Override
+                    public Object call() throws Exception {
+
+                        Calendar today = Calendar.getInstance();
+
+                        List<IPortletDefinition> portletList =
+                                portletDefinitionDao.getPortletDefinitions();
+                        List<ILocalAccountPerson> personList = localAccountDao.getAllAccounts();
+                        assertNotEquals(portletList.size(), 0);
+                        assertNotEquals(personList.size(), 0);
+                        int numberOfDaysBack = 1;
+                        for (IPortletDefinition portlet : portletList) {
+
+                            for (ILocalAccountPerson person : personList) {
+                                today.add(Calendar.DAY_OF_YEAR, 0 - numberOfDaysBack);
+                                System.out.println(
```

Comment:
:notebook: [`System.out.format`](https://docs.oracle.com/javase/tutorial/java/data/numberformat.html) may be able to help here

Reply:
Removed the println statement - it was not really needed